web/src/lib/http_client.ts

```diff
@@ -1,3 +1,3 @@
 import wretch from "wretch";
 
-export const httpClient = wretch("http://localhost:5173");
+export const httpClient = wretch("http://localhost:3000");
```

Comment:
I'd like to revert this change and the next one because i'd rather keep the proxy